i18n/de-DE.json

```diff
@@ -0,0 +1,127 @@
+        "warn": "<b>WARNING</b> Disabling the cache will delete all cached data.",
+        "info": "The cache stores all album artwork and other data to reduce the amount of requests to Apple. This will improve the loading time of the album artwork and link."
+      }
+    },
+    "extra": {
+      "github": "View on GitHub",
+      "ko-fi": "Donate",
+      "reloadPage": "Reload",
+      "restartApp": "Restart required"
+    },
+    "modal": {
+      "911": {
+        "title": "In memory of the victims of September 11",
+        "description": "To honor those who died on September 11 we added some custom album covers for these songs: %s\nAnd when you want, you can change all of the covers (only for today).",
+        "buttons": {
+          "okcover": "Okay, change the covers!"
+        }
+      },
+      "appleEvent": {
+        "title": "Do not forget!",
+        "description": "Today at %d o'clock (10a.m. PDT) is the Apple Event! Watch it live here: %s",
+        "buttons": {
+          "okcool": "Okay, cool!"
+        }
+      },
+      "newUpdate": {
+        "title": "Software Update",
+        "description": "Version %version% is available for your PC.",
+        "installed": {
+          "description": "Version %version% is available for your PC and is ready to install.",
+          "buttons": {
+            "install": "Install now",
+            "later": "Later"
+          }
+        },
+        "buttons": {
+          "downandinst": "Download & install",
+          "download": "Download & install later"
+        }
+      },
+      "alphaVersion": {
+        "title": "Alpha version",
+        "description": "Hello user! We are happy to announce that we have just released our alpha version of AMRPC. For now, we only have a few spots for the alpha version. So be quick if you want to test!",
+        "buttons": {
+          "register": "Register"
+        }
+      },
+      "amePlugin": {
+        "title": "Apple Music Electron Plugin",
+        "description": "The plugin was installed successfully.",
+        "descriptionError": "The plugin could not be installed."
+      },
+      "ko-fi": {
+        "title": "Requires Premium",
+        "description": "To use this feature, a one-time donation or a membership is required. You can donate <a href='https://ko-fi.com/zephra'>here</a>. If you have already donated, try to restart AMRPC. <a href='https://docs.amrpc.zephra.cloud/articles/ko-fi-features'>Learn More</a>"
+      },
+      "microsoftStore": {
+        "description": "Did you know that you can install AMRPC from the Microsoft Store? But make sure to disable <b>Auto Launch</b> in the settings before installing it from the Microsoft Store.",
+        "buttons": {
+          "download": "Download now"
+        }
+      },
+      "buttons": {
+        "no": "No",
+        "later": "Not now",
+        "yes": "Yes",
+        "okay": "Okay",
+        "learnMore": "Learn more",
+        "close": "Close"
+      }
+    }
+  }
+}
```

Comment:
```suggestion
{
  "error": {
    "admin": "Bitte starte AMRPC nicht mit Administrator-Rechten!",
    "noTrack": "No track detected"
  },
  "tray": {
    "restart": "AMRPC neu starten",
    "checkForUpdates": "Nach Updates suchen",
    "openSettings": "Einstellungen öffnen",
    "quit": "Beenden"
  },
  "settings": {
    "config": {
      "autoLaunch": "Autostart",
      "show": "Presence anzeigen",
      "hideOnPause": "Presence während Pause ausblenden",
      "showAlbumArtwork": "Album Artwork anzeigen",
      "performanceMode": "Performance Mode",
      "hardwareAcceleration": "Hardware-Beschleunigung",
      "colorTheme": {
        "label": "Bevorzugtes Farbschema",
        "light": "Hell",
        "dark": "Dunkel",
        "auto": "Dunkel von Sonnenuntergang bis -aufgang",
        "os": "System Farbschema"
      },
      "language": "Preferred Language",
      "cover": {
        "label": "Standard Artwork",
        "applemusic-logo": "Standard",
        "applemusic-logo-rainbow": "Regenbogen"
      },
      "service": "Service",
      "rpcDetails": "Presence Details",
      "rpcState": "Presence State",
      "enableCache": "Enable Cache"
    },
    "category": {
      "general": "Allgemein"
    },
    "warn": {
      "music": {
        "title": "Attention",
        "description": "The Music app could not be found. Please make sure that the Music app is installed."
      },
      "discord": {
        "title": "Attention",
        "description": "Discord could not be found. Please make sure that Discord is installed."
      }
    },
    "note": {
      "language": "If your language is missing, <a href='https://discord.gg/qqAgCHS'>contact us</a>.",
      "cache": {
        "size": "Current cache size: %size%, %count% items",
        "warn": "<b>WARNING</b> Disabling the cache will delete all cached data.",
        "info": "The cache stores all album artwork and other data to reduce the amount of requests to Apple. This will improve the loading time of the album artwork and link."
      }
    },
    "extra": {
      "github": "Auf GitHub ansehen",
      "ko-fi": "Spenden",
      "reloadPage": "Neu laden",
      "restartApp": "Neustart erforderlich"
    },
    "modal": {
      "911": {
        "title": "Zum Gedenken an die Opfer des 11. September",
        "description": "Zu Ehren der Opfer des 11. Septembers haben wir für diese Songs individuelle Albumcover erstellt: %s\nUnd wenn du möchtest, kannst du alle deine Covers ändern (nur für heute).",
        "buttons": {
          "okcover": "Okay, ändere die Covers!"
        }
      },
      "appleEvent": {
        "title": "Nicht vergessen!",
        "description": "Heute um %d Uhr (10.00 Uhr PDT) ist das Apple Event! Schaue es hier live: %s",
        "buttons": {
          "okcool": "Okay, cool!"
        }
      },
      "newUpdate": {
        "title": "Software Update",
        "description": "Die Version %version% ist für deinen PC verfügbar.",
        "installed": {
          "description": "Die Version %version% ist für deinen PC verfügbar und kann installiert werden.",
          "buttons": {
            "install": "Jetzt installieren",
            "later": "Später"
          }
        },
        "buttons": {
          "downandinst": "Laden & installieren",
          "download": "Laden & später installieren"
        }
      },
      "alphaVersion": {
        "title": "Alpha version",
        "description": "Hello user! We are happy to announce that we have just released our alpha version of AMRPC. For now, we only have a few spots for the alpha version. So be quick if you want to test!",
        "buttons": {
          "register": "Register"
        }
      },
      "amePlugin": {
        "title": "Apple Music Electron Plugin",
        "description": "Das Plugin wurde erfolgreich installiert.",
        "descriptionError": "Das Plugin konnte nicht installiert werden."
      },
      "ko-fi": {
        "title": "Erfordert Premium",
        "description": "Um dieses Feature nutzen zu können, ist eine one-time Spende oder Mitgliedschaft erforderlich. Du kannst <a href='https://ko-fi.com/zephra'>hier</a> spenden. Solltest du bereits gespendet haben, starte einmal AMRPC neu. <a href='https://docs.amrpc.zephra.cloud/articles/ko-fi-features'>Mehr erfahren</a>"
      },
      "microsoftStore": {
        "description": "Wusstest du, dass du AMRPC auch im Microsoft Store herunterladen kannst? Aber stelle sicher, dass du <b>Auto Launch</b> in den Einstellungen deaktivierst, bevor du AMRPC im Microsoft Store installierst.",
        "buttons": {
          "download": "Download now"
        }
      },
      "buttons": {
        "no": "Nein",
        "later": "Nicht jetzt",
        "yes": "Ja",
        "okay": "Okay",
        "learnMore": "Mehr erfahren",
        "close": "Schließen"
      }
    }
  }
}
```